.github/new-prs-labeler.yml

```diff
@@ -183,7 +184,8 @@ mlir:index:
   - mlir/**/Index/**
 
 mlir:llvm:
-  - mlir/**/LLVM/**
+  - mlir/**/LLVM*
+  - mlir/**/LLVM*/**
```

Comment:
Is there a difference between these two patterns for the labelled?

Reply:
Yes.  Here is the small js program I was using to check the patterns:
```
const { minimatch } = require('minimatch')

function check_pattern(file, pattern) {
    match = minimatch(file, pattern)
    console.log(file + " : " + pattern + " : " + match)
}
check_pattern('mlir/foo/LLVM.cpp', 'mlir/**/LLVM*')
check_pattern('mlir/foo/LLVM.cpp', 'mlir/**/LLVM*/**')
check_pattern('mlir/foo/LLVM/a.cpp', 'mlir/**/LLVM*')
check_pattern('mlir/foo/LLVM/a.cpp', 'mlir/**/LLVM*/**')
```
```
mlir/foo/LLVM.cpp : mlir/**/LLVM* : true
mlir/foo/LLVM.cpp : mlir/**/LLVM*/** : false
mlir/foo/LLVM/a.cpp : mlir/**/LLVM* : false
mlir/foo/LLVM/a.cpp : mlir/**/LLVM*/** : true
```